numpy/core/src/npymath/ieee754.c.src

```diff
@@ -330,21 +316,6 @@ static npy_longdouble _nextl(npy_longdouble x, int p)
  * Decorate all the math functions which are available on the current platform
  */
```

Comment:
Would be nice to remove the comment also.

Reply:
done, thanks